.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Which package needs legacy dependency? We should try to avoid it if possible, this might cause issues in future

Reply:
It has been created automatically with a storybook update. 
- deteled now

Reply:
Oh, I see it is causing error now. 

https://github.com/kbss-cvut/react-maintenance-planner/actions/runs/3677175025/jobs/6218782727

Reply:
After package-lock.json update, it seems working fine.

Reply:
Ok perfect, FYI you do not need webpack to make it run locally. Remove package-lock.json AND node_modules, resinstall everything with npm and should work :)